src/test/java/ladder/domain/DirectionTest.java

```diff
@@ -0,0 +1,34 @@
+package ladder.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class DirectionTest {
+    @ParameterizedTest(name = "{0}, {1}")
+    @CsvSource({
+            "false, true",
+            "true, false",
+            "false, false"
+    })
+    @DisplayName("방향 생성")
+    void create(boolean left, boolean right) {
+        // given
+        Direction direction = Direction.of(left, right);
+
+        // when
+        assertThat(direction.isLeft()).isEqualTo(left);
+        assertThat(direction.isRight()).isEqualTo(right);
+    }
```

Comment:
열거타입이니 직접 비교하면 어떨까요?
```suggestion
    @ParameterizedTest(name = "{0}, {1}")
    @CsvSource({
            "false, true, LEFT",
            "true, false, RIGHT",
            "false, false, NONE"
    })
    @DisplayName("방향 생성")
    void create(boolean left, boolean right, Direction expected) {
        // when
        Direction direction = Direction.of(left, right);

        // then
        assertThat(direction).isSameAs(expected);
    }
```

Reply:
오~ 감사합니다 ㅎㅎ